pkg/plugins/golang/v3/scaffolds/init.go

```diff
@@ -32,7 +32,7 @@ const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
 	ControllerRuntimeVersion = "v0.11.0"
 	// ControllerToolsVersion is the kubernetes-sigs/controller-tools version to be used in the project
-	ControllerToolsVersion = "v0.8.0"
+	ControllerToolsVersion = "v0.6.2"
```

Comment:
could you please revert this one?

Reply:
Yes, `controller-gen` v0.8.0 breaks `testdata/projectv3-v1beta1`. Need to figure out how to set only that one to an older version or fix the compatibility issues. See the PR description for more details.